WSEvents.cpp

```diff
@@ -642,6 +653,55 @@ void WSEvents::StreamStatus() {
     obs_output_release(stream_output);
 }
 
+void WSEvents::Heartbeat() {
+    bool streaming_active = obs_frontend_streaming_active();
+    bool recording_active = obs_frontend_recording_active();
+    obs_data_t* data = obs_data_create();
+
+    if (Heartbeat_active) {
+        pulse = !pulse;
+        obs_data_set_bool(data, "pulse", pulse);
```

Comment:
- This is the actual Heartbeat. It toggles every message.
- If the JSON message is every time the same, than you are not assured if things are alright. The toggling pulse shows that the host is alive.
- In NOCs (Network Operating Centers) the 24x7 monitoring systems do like to see this Heartbeat pulse. If toggle stops an alarm goes of. 
- I implemented in my Remote Control as a heart symbol that pulses:
![screen shot 08-19-17 at 10 23 pm](https://user-images.githubusercontent.com/30821653/29489972-7db42048-852e-11e7-8bac-48e046b37b9e.PNG)